support-lib/support_lib.gyp

```diff
@@ -43,5 +43,19 @@
                 ],
             },
         },
+        {
+            "target_name": "djinni_cx",
+            "type": "static_library",
+            "sources": [
+            ],
```

Comment:
Should list the header files, if nothing else.  Not sure if that's relevant to whichever generator you're using for this, but it's the normal standard used in the other targets in this file.